packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7000,31 +6971,40 @@ export default class Meeting extends StatelessWebexPlugin {
   }
 
   /**
-   * Un-publishes specified local tracks in the meeting
+   * Un-publishes specified local streams in the meeting
    *
-   * @param {Array<MediaStreamTrack>} tracks
+   * @param {Array<LocalStream>} streams
    * @returns {Promise}
    */
-  async unpublishTracks(tracks: LocalTrack[]): Promise<void> {
+  async unpublishStreams(streams: LocalStream[]): Promise<void> {
     this.checkMediaConnection();
 
     const promises = [];
+    let isAudioReleased = false;
```

Comment:
this should be named `isShareAudioReleased`, but overall the code using this variable doesn't seem right. What if share audio is in the `streams` list behind share video? or if the app calls unpublishStreams separately with just share audio and just share video?